4.0.0/migration.md

```diff
@@ -27,135 +146,148 @@ The migration process is initiated by the Installer if the following requirement
-         waitforpods.go:318] deployment "sysdigcloud-collector" successfully rolled out
-         waitforpods.go:315] Waiting for deployment "sysdigcloud-api" rollout to finish: 0 of 1 updated replicas are available...
-         waitforpods.go:315] Waiting for deployment "sysdigcloud-api" rollout to finish: 0 of 1 updated replicas are available...
-         waitforpods.go:318] deployment "sysdigcloud-api" successfully rolled out
-         waitforpods.go:74] waitforpods(sysdigcloud-collector,sysdigcloud-api) completed in 1m32s
-
-
-8.  Ensure that collector block for the new agent connections is disabled. To do so, run `Run GET /api/admin/agents/new/connections?block=false`
+   If scaling down is successful, you will see the following:
+
+        deploy.go:1427] scaling down alerter
+        deploy.go:1432] scaling down api
+        deploy.go:1447] scaling down worker
+        deploy.go:1476] removing secure cronjobs
+        deploy.go:1480] scaling down anchore
+        kubectl.go:216] deployments.apps "sysdigcloud-anchore-api" not found
+        kubectl.go:216] deployments.apps "sysdigcloud-anchore-catalog" not found
+        kubectl.go:216] deployments.apps "sysdigcloud-anchore-policy-engine" not found
+        deploy.go:1496] scaling down events
+        deploy.go:1501] scaling down policy-advisor
+        deploy.go:1504] scaling down scanning
+        deploy.go:1510] scaling down overview
+        deploy.go:1525] waiting pods to be terminated... (attempt 1/20)
+        deploy.go:1525] waiting pods to be terminated... (attempt 2/20)
+        deploy.go:1525] waiting pods to be terminated... (attempt 3/20)
+        deploy.go:1521] pods terminated correctly
+
+
+5. Ensure that the latest MySQL migrations are applied by the `mysql-latest-migrations` job:
+
+        deploy.go:1567] removing mysql latest migrations job
+        deploy.go:1570] running mysql latest migrations job
+        deploy.go:1583] waiting mysql latest migrations job to finish... (attempt 1/3600)
+        deploy.go:1583] waiting mysql latest migrations job to finish... (attempt 2/3600)
+        deploy.go:1583] waiting mysql latest migrations job to finish... (attempt 3/3600)
+        deploy.go:1580] mysql latest migrations finished correctly
+
+
+6. Ensure that MySQL databases are migrated to PostgreSQL by the mysql-to-postgresql-migration job:
+
+        deploy.go:1623] removing mysql to postgres migration job
+        deploy.go:1626] running mysql to postgres migration job
+        deploy.go:1685] waiting for mysql to postgres migration to finish... (attempt 1/3600)
+        deploy.go:1685] waiting for mysql to postgres migration to finish... (attempt 2/3600)
+        deploy.go:1636] mysql to postgres migration finished correctly
+
+
+7. Ensure that collector (running at this point) and the API server (not running at this point, as it was scaled it to 0 replicas earlier) are deployed using the new manifests pointing to PostgreSQL:
```

Comment:
Ensure that the collector (running at this point) and the API server (not running at this point, as it was scaled it to 0 replicas earlier) are deployed by using the new manifests pointing to PostgreSQL: